src/index.css

```diff
@@ -23,4 +23,48 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: rgb(176, 207, 176);
+}
+
+main {
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  margin: 0 auto;
+}
+
+.calculator {
+  display: grid;
+  grid-template-columns: repeat(4, 10rem);
+  grid-template-rows: minmax(7rem, auto) repeat(5, 6rem);
+  margin-top: 20px;
+}
+
+.header {
+  grid-column: 1 / -1;
+  background: #979393;
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-around;
+  flex-direction: column;
```

Comment:
- [ ] This is too large, reduce the height to match the given design
![image](https://github.com/Umuhan/calculator-app/assets/66097371/8b64d247-0986-40cb-a38c-51cf8aaefdcc)